### DISCUSSION TITLE
I need to trigger a event whenever changes in column picker in grid

### DISCUSSION THREAD

**Discussion by MohanramG** [ghiscoding/Angular-Slickgrid#914] [Q&A] [answered]
I have used (onColumnsChanged)="changed($event)" in html and changed(event:any) method in ts.But it is not triggering event and method is not getting called.
Any other changes i need to do? pls help me on this

**Comment by ghiscoding** [comment, marked as answer]
That event is to be used on the `columnPicker` and/or `gridMenu` grid options, you can't use it in the html it's not exposed as a grid event because it's not a grid thing, it's a column picker or grid menu thing so that's where you'll find it. You can see it in Example 9 (console log). I updated the [Column Picker - Wiki](https://github.com/ghiscoding/Angular-Slickgrid/wiki/Column-Picker) as well

https://github.com/ghiscoding/Angular-Slickgrid/blob/ee3bf0ec311ece5836234087668db6b6a9fe0dc7/src/app/examples/grid-menu.component.ts#L72-L78

Please consider asking your questions on Stack Overflow where more people can answer

**Reply by MohanramG** [reply]
Thanks for the information
